docs/concepts/models.md

```diff
@@ -1359,25 +1359,23 @@ print(error_locations)
 
 ## Required fields
```

Comment:
This is the updated documentation, the rest is just switching from `Field(...)` to `Field()`